python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -634,6 +637,40 @@ def get_two_hop_neighbors(self, start_vertices=None):
 
         return df
 
+    def select_random_vertices(self, random_state=None, num_vertices=None):
```

Comment:
Here as well, could you add specific tests for this function?

Reply:
Done